gap/main.gi

```diff
@@ -0,0 +1,67 @@
+# Install global functions of this package
+
+InstallGlobalFunction( DoEchelonMatTransformationBlockwise,
+    function ( mat, options... )
+    # options is a list that can optionally specify
+    # galoisField, IsHPC, numberChopsHeight, numberChopsWidth
+        local galoisField, IsHPC, numberChopsHeight, numberChopsWidth,
+            dim;
+
+        dim := DimensionsMat( mat );
+        if Size(options) < 4 then
+            numberChopsHeight := GAUSS_calculateChops( dim[1] );
+            numberChopsWidth := GAUSS_calculateChops( dim[2] );
+        else
+            numberChopsHeight := options[3];
+            numberChopsWidth := options[4];
+        fi;
+        Info(InfoGauss, 1, "The matrix has a height of ", dim[1],
```

Comment:
IMO we should only print the number of chops in both directions. The user already knows the size of the matrix. Unnecessary extra information tends to make the important information less visible.

Also please add an extra info statement that warns the user if any of `numberChops..` is equal to 1. In that case I think we will get almost no speed gain by using multiple threads.